tests/unit-tests/test_gnn.py

```diff
@@ -107,6 +107,12 @@ def check_node_prediction(model, data):
     assert_almost_equal(pred1[0:len(pred1)].numpy(), pred2[0:len(pred2)].numpy(), decimal=5)
     assert_equal(labels1.numpy(), labels2.numpy())
 
+    # Test the return_proba argument.
+    pred3, labels3 = node_mini_batch_predict(model, embs, dataloader1, return_proba=True, return_label=True)
+    assert pred3.dim() == 2  # returns all predictions (2D tensor) when return_proba is true
+    pred4, labels4 = node_mini_batch_predict(model, embs, dataloader1, return_proba=False, return_label=True)
+    assert(pred4.dim() == 1)  # returns maximum prediction (1D tensor) when return_proba is False
```

Comment:
Can we check the value of pred3 and pred4?

Reply:
Done.